Ratings_MVP/Ratings_MVPTests/Mocks/MockAssemblyModuleBuilder.swift

```diff
@@ -0,0 +1,18 @@
+// MockAssemblyModuleBuilder.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import Ratings_MVP
+import UIKit
+
+/// Mock AssemblyModuleBuilder
+final class MockAssemblyModuleBuilder: AssemblyBuilderProtocol {
+    // MARK: - Public Methods
+    
+    func makeMoviesListScreen(router: MoviesRouterProtocol) -> UIViewController {
+        MoviesListViewController()
```

Comment:
```suggestion
        UIViewController()
```
А чего ты просто контроллер не вернешь?)